tests/system/feature_store/test_feature_store.py

```diff
@@ -2994,6 +2994,165 @@ def test_alias_change(self):
         finally:
             service.close()
 
+    @pytest.mark.parametrize("engine", ["local", "dask"])
+    def test_relation_join(self, engine):
```

Comment:
add a small doc string like:

"""
Testing the X with Y. Expected result: Z
"""